client/src/Clusters/RecommendedCategories/card_container_style.css

```diff
@@ -0,0 +1,21 @@
+.card {
+    width: calc(25% - 20px);
+    margin: 10px;
+}
+
+.box {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.img-section {
+    float: right;
+    height: 60px;
+    width: 60px;
+}
+
+.text-section {
+    float: left;
+    font-size: 25px;
```

Comment:
Let's true to use `rem` instead of `px`. It makes it easier to make the font size more responsive later.